rmltraintfbbox/rmltraintfbbox/validation/stats.py

```diff
@@ -53,7 +54,7 @@ def load_from_dump(cls, dump_path):
         self.stats = {}
         return self
 
-    def add_single_result(self, output, inference_time, bbox, centroid, image_size=None, distance=None):
+    def add_single_result(self, output, true_shape, inference_time, bbox, centroid, image_size=None, distance=None):
```

Comment:
Could you add a docstring comment for `true_shape`?

Reply:
Done, thanks for the catch.